packages/pangea-sdk/pangea/asyncio/services/vault.py

```diff
@@ -129,120 +142,118 @@ def __init__(
         """
         super().__init__(token, config, logger_name)
 
-    # Delete endpoint
-    async def delete(self, id: str) -> PangeaResponse[DeleteResult]:
+    async def delete(self, item_id: str, *, recursive: bool = False) -> PangeaResponse[DeleteResult]:
         """
         Delete
 
         Delete a secret or key
 
-        OperationId: vault_post_v1_delete
+        OperationId: vault_post_v2beta_delete
```

Comment:
operation should be updated to v2 as same as paths

Reply:
Yup, that wasn't available at the time. Done.